WcaOnRails/app/models/paypal_transaction.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class PaypalTransaction < ApplicationRecord
+  has_many :paypal_captures
+
+  # NOTE: Is this even necessary? We could just use positive/negative amounts to differentiate them?
+  TRANSACTION_TYPES = [
+    :payment,
+    :refund,
+  ].freeze
+
+  # Defined in: https://developer.paypal.com/docs/reports/reference/paypal-supported-currencies/
+  PAYPAL_CURRENCY_CATEGORIES = {
+    decimal: [ # Currencies that should be passed to paypal as decimal amounts (ie, cents/100)
+      "AUD",
+      "BRL",
+      "CAD",
+      "CNY",
+      "CZK",
+      "DKK",
+      "EUR",
+      "HKD",
+      "ILS",
+      "MYR",
+      "MXN",
+      "NZD",
+      "NOK",
+      "PHP",
+      "PLN",
+      "GBP",
+      "SGD",
+      "SEK",
+      "CHF",
+      "THB",
+      "USD",
+    ],
+    cents_only: [ # Currencies that do not support decimals - should be passed as cents
+      "JPY",
+      "HUF",
+      "TWD",
+    ],
+  }.freeze
+
+  def self.get_paypal_amount(amount_in_cents, currency_code)
```

Comment:
Naming convention: `def paypal_amount`. No `get_` in Ruby because... Ruby.

Reply:
Didn't know that, thanks!